src/Stratis.Bitcoin.Features.RPC/Controllers/FullNodeController.cs

```diff
@@ -82,7 +83,10 @@ public Task Stop()
             if (!uint256.TryParse(txid, out trxid))
                 throw new ArgumentException(nameof(txid));
 
-            Transaction trx = this.pooledTransaction != null? await this.pooledTransaction.GetTransaction(trxid) : null;
+            Transaction trx = this.pooledTransaction != null ? await this.pooledTransaction.GetTransaction(trxid) : null;
+
+            if (this.pooledTransaction != null)
```

Comment:
this code seems to be duplicated.

Reply:
I will have a look. It seem so to me as well.